recipes/git-annex-remote-googledrive/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "git-annex-remote-googledrive" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "8d3edcd9cbcf7c7f5617eaacd14f275ed6b5cbafd3e66f4f3a0f75d86b675eed"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k or not linux]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - annexremote
+    - pydrive
+    - python
```

If this is made `noarch`:
```suggestion
    - python >=3
```